website/docs/docs/quickstarts/overview.md

```diff
@@ -0,0 +1,52 @@
+---
+title: About quickstarts
+id: overview
+sidebar_label: "About quickstarts"
+description: "Quickly create your first dbt project with SQL."
+---
+<section className="community-home">
+
+Learn how to quickly start using dbt Cloud or dbt Core using a sample project and one of the most common data platforms. The quickstarts will show you how to set up dbt and perform some key tasks. These tasks will illustrate how dbt establishes standard practices for your work such as using Git to version control your analytics code, testing your models to validate they are working correctly, documenting your models in detail to better collaborate with others, and scheduling jobs to execute them at your designated times.  
+
+Before you begin, you will need:
+
+* Warehouse with sample data. If you don't have this, you can use the [BigQuery project](/docs/quickstarts/dbt-cloud/bigquery), which leverages public data sets.
+* Basic understanding of Git.
+* Basic understanding of SQL.
+
+
+<div className="grid--2-col">
+
+<Card
+    title="Quickstart for dbt Cloud and BigQuery"
+    body="dbt Cloud is the fastest and most reliable way to deploy dbt. Learn how to connect to BigQuery, build your first dbt models with a sample project, test your models, document your models, and schedule a job run all in one web-based UI."
+    link="/docs/quickstarts/dbt-cloud/bigquery"
+    icon="bigquery"/>
+
+<Card
+    title="Quickstart for dbt Cloud and Databricks"
+    body="dbt Cloud is the fastest and most reliable way to deploy dbt. Learn how to connect to Databricks, build your first dbt models with a sample project, test your models, document your models, and schedule a job run all in one web-based UI."
+    link="/docs/quickstarts/dbt-cloud/databricks"
+    icon="databricks"/>
+
+<Card
+    title="Quickstart for dbt Cloud and Redshift"
+    body="dbt Cloud is the fastest and most reliable way to deploy dbt. Learn how to connect to Redshift, build your first dbt models with a sample project, test your models, document your models, and schedule a job run all in one web-based UI."
+    link="/docs/quickstarts/dbt-cloud/redshift"
+    icon="redshift"/>
+
+<Card
+    title="Quickstart for dbt Cloud and Snowflake"
+    body="dbt Cloud is the fastest and most reliable way to deploy dbt. Learn how to connect to Snowflake, build your first dbt models with a sample project, test your models, document your models, and schedule a job run all in one web-based UI."
+    link="/docs/quickstarts/dbt-cloud/snowflake"
+    icon="snowflake"/>
+
+<Card
+    title="Quickstart for dbt Core"
+    body="When you use dbt Core to work with dbt, you will be editing files locally using a code editor, and running projects using a command line interface."
+    link="/docs/quickstarts/dbt-core/quickstart"
+    icon="pencil-paper"/>
```

Comment:
Would be cool if there was a specialist icon here too, e.g. https://fontawesome.com/icons/square-terminal?s=solid&f=classic